released/index.yaml

```
@@ -3,7 +3,7 @@ entries:
   talend-api-microgateway:
   - apiVersion: v1
     appVersion: "1.4"
-    created: "2020-01-15T11:57:14.023987+01:00"
+    created: "2020-05-18T22:45:44.394551+02:00"
```

```suggestion
    created: "2020-01-15T11:57:14.023987+01:00"
```
I would suggest to keep the date if the content/version does not change